UPGRADE.md

```diff
@@ -14,6 +14,11 @@
 
 ### Core / CoreBundle
 
+* `Sylius\Component\Core\OrderProcessing\OrderTaxesProcessor` and `Sylius\Component\Core\Resolver\ZoneAndChannelBasedShippingMethodsResolver` 
+have become a zone scope aware. From now, only zones with scope 'shipping' will be considered in `Sylius\Component\Core\Resolver\ZoneAndChannelBasedShippingMethodsResolver` 
+and a scope 'tax' is required in `Sylius\Component\Core\OrderProcessing\OrderTaxesProcessor`. 
```

Comment:
The above is not valid, the services will now match all zones with scope `tax/shipping` and also all zones with scope `all`.